stories/TokenOverlay.tsx

```diff
@@ -2,17 +2,16 @@ import * as React from 'react'
 
 import { storiesOf } from '@storybook/react'
 import { array, object, boolean } from '@storybook/addon-knobs'
-import { action } from '@storybook/addon-actions'
+import { action, decorateAction } from '@storybook/addon-actions'
 
 import TokenOverlay from 'components/TokenOverlay'
+import { TokenBalances } from 'types'
 
-import { code2tokenMap, TokenCode } from 'globals'
-
-const codeList = Object.keys(code2tokenMap) as TokenCode[]
+import { codeList } from 'globals'
 
 const tokenBalances = codeList.reduce(
   (acc, code) => (acc[code] = (Math.random() * 5).toFixed(9), acc), {},
-) as {[code in TokenCode]: number }
+) as TokenBalances
 
 const CenterDecor = (story: Function) => (
```

Comment:
Do you not want to follow the @storyDecorator syntax here and below?

Reply:
Missed it, will change